i18n/qqq.json

```diff
@@ -1,11 +1,18 @@
 {
 	"@metadata": {
 		"authors": [
-			"Kghbln",
-			"Shirayuki",
-			"Андрей"
+			"MWJames",
+			"Kghbln"
 		]
 	},
-	"sdata-desc": "{{desc|name=Semantic Extra Special Properties|url=https://www.mediawiki.org/wiki/Extension:Semantic_Extra_Special_Properties}}",
-	"sdata-name": "{{notranslate}}\nName of the extension as it should be displayed."
+	"semantic-approvedrevs-desc": "{{desc|name=Semantic Approved Revs|url=hhttps://www.mediawiki.org/wiki/Extension:Semantic_Approved_Revs}}",
+	"semantic-approvedrevs-name": "{{notranslate}}\nSemantic Approved Revs"
+	"semantic-approvedrevs-property-approved-rev": "Approved revision",
```

Comment:
Add something like "The name of the special property." You can use SESP qqq file as an example here. Same for the other lot.

Reply:
ah, I see. Thank you for the explanation!